strapi/strapi
# Add icon picker to components and show component's icons

### What does it do?

Last year we removed FontAwesome dependency on #15133 , on that PR we remove the icons from components on Content Manager and CTB because they were using FontAwesome. With this PR we add again the icon picker on components as well as the icon when you are creating a new entry with a Dynamic Zone, but this time using our own components from the Design System and not FontAwesome

### How to test it?

- **CTB:** Go to any content type and add a new field of type component, you should see a new icon picker where you can select the icon you want
- **CM:** When creating an entry with a dynamic zone, you should see the icon for each component, you can see also the icons at the "Configure the view" page
